.github/integration/tests/common/30_ingest_test.sh

```diff
@@ -2,7 +2,7 @@
 
 docker run --rm --name client --network dev_utils_default \
 neicnordic/pg-client:latest postgresql://lega_out:lega_out@db:5432/lega \
--t -c "SELECT * from local_ega_ebi.file_dataset ORDER BY id DESC"
+-t -c "SELECT * from local_ega_ebi.file_dataset"
```

Comment:
Why remove the sorting?
With the `10_trigger_failures` script running it is nice to see that it is really the last two inserted files that gets the the Accession and dataset ids

Reply:
The view local_ega_ebi.file_dataset does not have any `id` column, if desirable we can sort on one of the ids available.